common/js/src/messaging/message-channel-pool.js

```diff
@@ -16,16 +16,17 @@
 
 /**
  * @fileoverview This file containts the pool that keeps track of all the
- * Closure-style AbstractChannel's used for messaging
- * (PortMessageChannel & SingleMesageBasedChannel).
+ * Closure-style AbstractChannel's used for messaging (PortMessageChannel &
```

Comment:
Quoting the exact classes names ("PortMessageChannel", "SingleMesageBasedChannel") is unnecessary here. This class works regardless of the concrete element types.